lib/pages/timetable/view/events/all_day_event_widget.dart

```diff
@@ -1,248 +1,249 @@
-//
-//  @override
-//  String toString() =>
-//      '${objectRuntimeType(this, 'RoundedRectangleBorder')}($side, $borderRadius)';
-//}
-//
-//Path _getPath(Size size, AllDayEventLayoutInfo info, double radius) {
-//  final height = size.height;
-//  // final radius = borderRadius.coerceAtMost(width / 2);
-//
-//  final maxTipWidth = height / 4;
-//  final leftTipWidth = info.hiddenStartDays.coerceAtMost(1) * maxTipWidth;
-//  final rightTipWidth = info.hiddenEndDays.coerceAtMost(1) * maxTipWidth;
-//
-//  final width = size.width;
-//  // final leftTipBase = math.min(leftTipWidth + radius, width - radius);
-//  // final rightTipBase = math.max(width - rightTipWidth - radius, radius);
-//  final leftTipBase = info.hiddenStartDays > 0
-//      ? math.min(leftTipWidth + radius, width - radius)
-//      : leftTipWidth + radius;
-//  final rightTipBase = info.hiddenEndDays > 0
-//      ? math.max(width - rightTipWidth - radius, radius)
-//      : width - rightTipWidth - radius;
-//
-//  final tipSize = Size.square(radius * 2);
-//
-//  // no tip:   0      ≈  0°
-//  // full tip: PI / 4 ≈ 45°
-//  final leftTipAngle = math.pi / 2 - math.atan2(height / 2, leftTipWidth);
-//  final rightTipAngle = math.pi / 2 - math.atan2(height / 2, rightTipWidth);
-//
-//  return Path()
-//    ..moveTo(leftTipBase, 0)
-//    // Right top
-//    ..arcTo(
-//      Offset(rightTipBase - radius, 0) & tipSize,
-//      math.pi * 3 / 2,
-//      math.pi / 2 - rightTipAngle,
-//      false,
-//    )
-//    // Right tip
-//    ..arcTo(
-//      Offset(rightTipBase + rightTipWidth - radius, height / 2 - radius) &
-//          tipSize,
-//      -rightTipAngle,
-//      2 * rightTipAngle,
-//      false,
-//    )
-//    // Right bottom
-//    ..arcTo(
-//      Offset(rightTipBase - radius, height - radius * 2) & tipSize,
-//      rightTipAngle,
-//      math.pi / 2 - rightTipAngle,
-//      false,
-//    )
-//    // Left bottom
-//    ..arcTo(
-//      Offset(leftTipBase - radius, height - radius * 2) & tipSize,
-//      math.pi / 2,
-//      math.pi / 2 - leftTipAngle,
-//      false,
-//    )
-//    // Left tip
-//    ..arcTo(
-//      Offset(leftTipBase - leftTipWidth - radius, height / 2 - radius) &
-//          tipSize,
-//      math.pi - leftTipAngle,
-//      2 * leftTipAngle,
-//      false,
-//    )
-//    // Left top
-//    ..arcTo(
-//      Offset(leftTipBase - radius, 0) & tipSize,
-//      math.pi + leftTipAngle,
-//      math.pi / 2 - leftTipAngle,
-//      false,
-//    );
-//}
+import 'dart:math' as math;
+
+import 'package:black_hole_flutter/black_hole_flutter.dart';
+import 'package:dartx/dartx.dart';
+import 'package:flutter/foundation.dart';
+import 'package:flutter/material.dart';
+import 'package:timetable/timetable.dart';
+
+import '../../model/events/uni_event.dart';
+import 'event_view.dart';
+
+/// Widget to display all day events in the timetable, based on
+/// [BasicAllDayEventWidget] from the timetable API.
```

Comment:
Q: Did this widget that our implementation is based on change significantly with the new timetable version? We may want to consider updating our code here.

Reply:
Slightly https://www.diffchecker.com/F8RyIUK4